### PR TITLE
Added typings for react-image-fallback package

### PR DESCRIPTION
**Pull request by 8enSmith**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/react-image-fallback/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    React.DetailedHTMLProps<
+    React.ImgHTMLAttributes<HTMLImageElement>,
+    HTMLImageElement
+    >,
+    any
+    > { }
+
+export interface ReactImageFallbackProps {
+    src: string;
+    fallbackImage: string | React.ReactElement<any> | any[];
```
Comment (SimonSchick) [inline types/react-image-fallback/index.d.ts]: Can you please narrow `any[]` here?

Comment (8enSmith) [reply types/react-image-fallback/index.d.ts]: As per the [documentation](https://github.com/socialtables/react-image-fallback#fallbackimage), fallbackImage should be a string or a React element. As it's not specific with regards to what type of ReactElement it should be I've used 'any'. Suggestions?

Comment (8enSmith) [reply types/react-image-fallback/index.d.ts]: Ah yes, I'll get this fixed ASAP.

Comment (8enSmith) [reply types/react-image-fallback/index.d.ts]: I've pushed a change with more specific typings as per the package [documentation](https://github.com/socialtables/react-image-fallback#fallbackimage).

**Comment by 8enSmith** [comment]
@SimonSchick This PR now passes the Travis review. As this is my first DefinetlyTyped PR not sure if you do a review or I wait it to go through the backlog?

**Review by SimonSchick — APPROVED** [review]

**Comment by sandersn** [comment]
Thanks @8enSmith !

**Comment by 8enSmith** [comment]
Hi @RyanCavanaugh & @andy-ms . This PR was merged into master 2 days ago but it's still not showing up in https://microsoft.github.io/TypeSearch/. Has something gone wrong or is this delay expected?

**Comment by 8enSmith** [comment]
> Hi @RyanCavanaugh & @andy-ms . This PR was merged into master 2 days ago but it's still not showing up in https://microsoft.github.io/TypeSearch/. Has something gone wrong or is this delay expected?

Ah I can see that it is present on [npmjs.com](https://www.npmjs.com/package/@types/react-image-fallback) but it's not yet showing up on  https://microsoft.github.io/TypeSearch/.

**Pull request merged**